src/cincinnati/mod.rs

```diff
@@ -277,7 +277,13 @@ fn evaluate_deadend(node: &Node) -> Option<String> {
     if deadend_reason.is_empty() {
         deadend_reason = "(unknown reason)".to_string();
     }
-
+    if !deadend_reason.is_empty() {
+        let _ = std::fs::remove_file("/etc/motd");
+        let data = format!("version: {} is a dead-end release", node.version);
```

Comment:
The version here is not very interesting. The important bits are:
 * is this node running on a deadend release (boolean, required)
 * if so, what's the reason/detail for this deadend release (string, optional)